networkit/cpp/io/GMLGraphReader.cpp

```diff
@@ -69,7 +69,7 @@ Graph GMLGraphReader::read(const std::string& path) {
 						nodeMap.insert(std::make_pair(pair.second,u));
 						DEBUG("added node: ",u,", ",pair.second);
 					}	
-				} catch (std::exception e) {
+				} catch (std::exception &e) {
```

Comment:
We should catch by `const` reference, unless we need to modify the exception (which would be pretty strange).